meta-resin-common/recipes-core/initrdscripts/files/finish

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
Why do we have to inject code between mounting the rootfs and switching to it? What code? This kind of info should be in the commit log. Without it, I don't really understand the need for splitting this